### PR TITLE
repos: Update for scarthgap

### PR DESCRIPTION
**Pull request by chaitu236**
Some repos now have scarthgap branches so update those.

WI: [AB#2468923](https://dev.azure.com/ni/DevCentral/_workitems/edit/2468923)

### Testing
- [x] Ran `upstream_merge.sh` script
- [x] bitbake packagefeed-ni-core
- [x] bitbake packagegroup-ni-desirable
- [x] bitbake package-index && bitbake nilrt-base-system-image
- [x] Installed BSI on a VM and verified it boots

### CODE REVIEW

File: scripts/dev/upstream_merge/repos.conf [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+sources/bitbake              https://git.openembedded.org/bitbake                  2.8                   nilrt/master/next
 sources/meta-cloud-services  https://git.yoctoproject.org/meta-cloud-services      master                nilrt/master/next
-sources/meta-openembedded    https://git.openembedded.org/meta-openembedded        master                nilrt/master/next
-sources/meta-selinux         https://git.yoctoproject.org/meta-selinux             master                nilrt/master/next
-sources/openembedded-core    https://git.openembedded.org/openembedded-core        master                nilrt/master/next
+sources/meta-openembedded    https://git.openembedded.org/meta-openembedded        scarthgap             nilrt/master/next
+sources/meta-selinux         https://git.yoctoproject.org/meta-selinux             scarthgap             nilrt/master/next
+sources/openembedded-core    https://git.openembedded.org/openembedded-core        scarthgap             nilrt/master/next
 sources/meta-virtualization  https://git.yoctoproject.org/meta-virtualization      master                nilrt/master/next
```
Comment (gratian) [inline scripts/dev/upstream_merge/repos.conf]: FWIW meta-virtualization also just sprouted a 'scarthgap' branch. It's probably worth checking the other ones below too.

File: scripts/dev/upstream_merge/repos.conf [hunk 2]
```diff
@@ -1,14 +1,14 @@
 #local_repo                  upstream_repo                                         upstream_branch       local_base_branch
-sources/bitbake              https://git.openembedded.org/bitbake                  master                nilrt/master/next
+sources/bitbake              https://git.openembedded.org/bitbake                  2.8                   nilrt/master/next
 sources/meta-cloud-services  https://git.yoctoproject.org/meta-cloud-services      master                nilrt/master/next
```
Comment (gratian) [inline scripts/dev/upstream_merge/repos.conf]: Looks like meta-cloud-services has a scarthgap branch now.

File: scripts/dev/upstream_merge/repos.conf (hunk 1 above)
Comment (chaitu236) [reply scripts/dev/upstream_merge/repos.conf]: Thanks for pointing this out. They must have merged `master-next` to `master` and created `scarthgap` branch today sometime after I ran `upstream_merge.sh` as there are newer commits in upstream's `master` than when I ran the script.

File: scripts/dev/upstream_merge/repos.conf (hunk 2 above)
Comment (chaitu236) [reply scripts/dev/upstream_merge/repos.conf]: Thanks. Updated

**Review by gratian — APPROVED** [review]

**Review by amstewart — APPROVED** [review]

**Pull request merged**